packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import * as React from "react";
+
+import FieldHeading from "./field-heading.js";
+import TextField from "./text-field.js";
+
+export default class LabeledTextField extends React.Component<{||}> {
+    render(): React.Node {
+        return (
+            <FieldHeading
+                field={
+                    <TextField
+                        id="tf-1"
+                        value="Value"
+                        placeholder="Placeholder"
+                        onChange={() => {}}
+                    />
+                }
+                label="Label"
+                description="Description"
+                error="Error"
+            />
+        );
+    }
+}
```

Comment:
This component could be a functional stateless component. Would be less indentation - just an option. No change required.

```js
export default (props: Props) => (
    <FieldHeading... etc. etc.
);
```